src/main/wrapper/ExecutionService.ts

```diff
@@ -24,6 +24,8 @@ import CxChat from "../chat/CxChat";
 import CxMask from "../mask/CxMask";
 import CxVorpal from "../vorpal/CxVorpal";
 
+let skipValue = false;
```

Comment:
Can you make skipValue variable of transofrmation function and pass it as a parameter to transform? I think it's better practice then global variable

Reply:
We will insert it later so as not to delay the release